config/array_flags.go

```diff
@@ -1,51 +0,0 @@
-// Copyright 2019 the orbs-network-go authors
```

Comment:
Please bring it back, it's a useful feature.

Reply:
It was not deleted. it was moved to another file and re-named because ArrayFlags means nothing and names are important. (look for it in config_modifiers.go

Reply:
also in my opinion (but i did not delete it yet) allowing running ONG with multiple json config files is NOT a useful feature, more like a hack.

Reply:
It is useful from deployment perspective because it allows to combine and share configuration both in e2e and production.

Although in production when everything moves to management file we can consider removing it completely.